crates/biome_cli/tests/commands/check.rs

```diff
@@ -2797,3 +2797,52 @@ fn use_literal_keys_should_emit_correct_ast_issue_266() {
         result,
     ));
 }
+
+#[test]
+fn should_show_diagnostics_for_linter_ignored_file_when_folder_is_ignored() {
```

Comment:
As a user, I would expect the files inside any `build` dir to be ignored if I configured `"ignore": ["**/build"]` in the `linter` section. So it shouldn't show any diagnostics for the files inside any `build` dir.

Reply:
Ah my bad, I poorly named the test file. Good catch. It should be formatter diagnostics, NOT linter diagnostics

Reply:
~~But the snapshot shows the linter diagnostics are printed.~~ Yeah you found it :)